### ISSUE TITLE
Merge with timetable package

### ISSUE THREAD

**Issue by JulianKowalczuk** [Skyost/FlutterWeekView#49]
Hello,

by browsing Flutter calendar packages I found out this one:
https://pub.dev/packages/timetable

Since it's more customizable (lack of this feature was the reason I dropped using Your one) it would be nice to contact with its author and maybe to team up.
The only disadvantage of JonasWanke's package is time_machine package usage (which is forbidden by its author, but I created issue for it).

**Comment by Skyost** [comment]
If @JonasWanke wants some help for his package (or if he wants to contribute to this one), I will be happy to talk with him. But for now there will be no merging because I created FlutterWeekView for me at first, as a mean to only get a WeekView and a DayView. Something I could easily update and maintain.

By sharing it, I just wanted to give to the community the same thing I was searching for : a lightweight package that is able to display a completely customizable Day / Week view ; nothing more. And as [timetable](https://pub.dev/packages/timetable) aims to do more and different things, we have two different goals.

To sum up :

* Two different authors.
* Two different goals.
* Two different code bases.

→ Two different packages.
